src/Features/SupportLockedProperties/CannotUpdateLockedPropertyException.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Livewire\Features\SupportLockedProperties;
+
+class CannotUpdateLockedPropertyException extends \Exception
+{
+    public function __construct($property)
```

Comment:
is it possible to do `__construct(public $property)` to store the property in a way that it can be used without parsing the exception message afterwards?

Reply:
Done!